ansible/pbTestScripts/qemu_test_script.sh

```diff
@@ -120,7 +129,7 @@ local workFolder="$WORKSPACE/qemu_pbCheck"
 
 # Find/stop port collisions
 # while ps -aux | grep "$PORTNO" | grep -q -v "grep"; do
-while netstat -lp 2>/dev/null | grep "tcp.*:$PORTNO " > /dev/nulln; do
+while netstat -lp 2>/dev/nulln | grep "tcp.*:$PORTNO " > /dev/null; do
```

Comment:
nulln? You've switched them around but that seems wrong regardless

Reply:
typo- my bad